frontend/src/views/containers/game-window/GameWindow.tsx

```diff
@@ -1,11 +1,30 @@
 /* eslint-disable no-restricted-imports */
-import React from "react";
+import React, { useState } from "react";
 import { ViewGame } from "../../../controllers/models/ViewGame";
+import { ViewGameInProgress } from "../../../controllers/models/ViewGameInProgress";
+import { LetterKeyboard } from "../letter-keyboard/LetterKeyboard";
 
 interface Props {
   game: ViewGame;
 }
 
 export const GameWindow = ({ game }: Props) => {
-  return <div data-testid="player-id">{game.playerId}</div>;
+  const [newGame, setGame] = useState<ViewGameInProgress>();
+
+  const lettersGuessed = newGame === undefined ? [] : newGame.lettersGuessed;
+
+  return (
+    <div>
+      {newGame === undefined ? (
```

Comment:
Nelabai man ir šita vieta patinka. Turėtų būti Single Source Of Truth, o čia turi kažkokį newGame ir tada game. Kuo jie skiriasi? Ar sukūrus žaidimą state set'ina į abu state?